src/Arcus.Template.WebApi/Program.cs

```diff
@@ -15,7 +15,10 @@ public static void Main(string[] args)
 
         public static IWebHostBuilder CreateWebHostBuilder(string[] args)
         {
+            const string httpEndpointUrl = "http://+:5000";
```

Comment:
Just a suggestion, not sure but do I have to retrieve this from the configuration?

Reply:
If we use this, then it has to be passed via an environment variable.

This is more of a deployment thingy than fixed imo, no?

Reply:
In terms of looking it up you can directly call environment variable or create default config provider and pass it to the builder as well.

Adding default configuration with environment variables would be good I think, no?

Reply:
Done